samples/node/src/connectedcarsapi.js

```diff
@@ -56,15 +63,18 @@ class ConnectedCarsApi {
    * @param {string} ccServiceAccountKeyData a string containing the Connected Cars service account data
    * @param {string} [endpoint] specify the connected cars api endpoint, default is production endpoint
    * @param {string} [authEndpoint] specify the connected cars auth endpoint, default is production endpoint
+   * @param {string} [organizationNamespace] specify the organization namespace, ask Connected Cars for more info
    * @throws an error if the service account data is malformed
    */
   constructor(
     ccServiceAccountKeyData,
     endpoint = 'https://api.connectedcars.io/graphql',
-    authEndpoint = 'https://auth-api.connectedcars.io/auth/login/serviceAccountConverter'
+    authEndpoint = 'https://auth-api.connectedcars.io/auth/login/serviceAccountConverter',
+    organizationNamespace = 'semler:app'
```

Comment:
I'm not sure if it is okay, that we expose this as a default value...?

Reply:
Fixed